projects/website/src/app/documentation/demos/datagrid/datagrid.demo.main.scss

```diff
@@ -11,3 +11,18 @@
   column-count: 3;
   list-style-position: inside;
 }
+
+::ng-deep.clr-accordion:not(.clr-stepper-forms) .clr-accordion-inner-content {
```

Comment:
It would be a lot simpler to make the padding/margins inside accordion if we were using core here. Alas I resorted to `::ng-deep` to override the interal accordion styles.